<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_camera_menu</class>
 <widget class="QMainWindow" name="interface_camera_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget_camera_menu{background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(114, 140, 153, 255), stop:1 rgba(0, 39, 59, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget_camera_menu">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_work" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>500</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QTextEdit" name="text_edit_name_image">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;_&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QTextEdit" name="text_edit_exposure_gain">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Exposure&amp;amp;Gain&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_image_name">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3" colspan="4">
        <widget class="QWidget" name="widget_for_stream" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QGridLayout" name="companovka_stream"/>
          </item>
          <item>
           <widget class="QLabel" name="label_cords_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9999</width>
              <height>13</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>x,y,z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_tiff">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>.tiff</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="4" colspan="2">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>500</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_camera_setting">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <attribute name="title">
           <string>Camera setting</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <item row="1" column="1">
            <spacer name="verticalspacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="13" column="0" colspan="3">
            <widget class="QLabel" name="label_exposure">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Exposure [ms]:</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="3">
            <widget class="QLabel" name="label_gain">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Gain:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="3">
            <spacer name="verticalspacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="19" column="0" colspan="3">
            <widget class="QLabel" name="label_fps">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Fps:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="3">
            <widget class="QCheckBox" name="checkbox_gray_color">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Gray / Color</string>
             </property>
            </widget>
           </item>
           <item row="21" column="0" colspan="3">
            <widget class="QDoubleSpinBox" name="spinbox_fps">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>2.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="18" column="0" colspan="3">
            <widget class="QSpinBox" name="spinbox_gain">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>33</number>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="3">
            <widget class="QCheckBox" name="checkbox_grid_stream">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Grid</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="button_start_stream">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>72</width>
               <height>72</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>72</width>
               <height>72</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>1</width>
               <height>1</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/icon/icon/baseline_play_circle_outline_black_36dp.png);
</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="3">
            <widget class="QDoubleSpinBox" name="spinbox_exposure">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>60000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
             <property name="value">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QWidget" name="widget_4" native="true">
             <property name="font">
              <font>
               <italic>false</italic>
              </font>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QPushButton" name="button_posun_stream">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_pan_tool_black_24dp.png);</string>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="shortcut">
                 <string/>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="button_zoom_stream">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_search_black_24dp.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="button_up_to_stream">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_arrow_back_black_24dp.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="button_save_image_stream">
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_save_black_36dp.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="button_home_stream">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_account_balance_black_24dp.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="button_down_to_stream">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/icon/icon/baseline_arrow_forward_black_24dp.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="10" column="0" colspan="3">
            <widget class="QCheckBox" name="checkbox_axis">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>Axis</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_analysis">
          <attribute name="title">
           <string>Analysis</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QPushButton" name="button_open_analysis_window">
             <property name="text">
              <string>Open Analysis window</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkbox_line_profile">
             <property name="text">
              <string>Line profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkbox_select_area">
             <property name="text">
              <string>Select area</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_select_data_from_area">
             <property name="text">
              <string>Select data from area</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_homogeneity">
             <property name="text">
              <string>Homogeneity</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalspacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_catch_reference">
             <property name="text">
              <string>Catch reference</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkbox_implement_reference">
             <property name="text">
              <string>Implement reference</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource/image_resource.qrc"/>
  <include location="resource/icon_resource.qrc"/>
 </resources>
 <connections/>
</ui>
